config_panel.toml

```diff
@@ -0,0 +1,16 @@
+version = "1.0"
+
+[bind]
+name = "Mautrix Signal configuration"
+
+    [bind.encryption]
+    name = "Manage encryption"
+
+        [bind.encryption.encryption]
```

Comment:
Shouldn't this be `bind.encryption.encryption_allowed` to be more meaningful?

Also this should reuse the name while using `ynh_app_setting_set`.